drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -456,6 +470,7 @@ local function device_added(driver, device)
   device:send(clusters.Thermostat.attributes.ControlSequenceOfOperation:read(device))
   device:send(clusters.FanControl.attributes.FanModeSequence:read(device))
   device:send(clusters.FanControl.attributes.WindSupport:read(device))
+  device:send(clusters.FanControl.attributes.RockSupport:read(device))
```

Comment:
I know this existed before your changes, but one possible improvement is to merge these reads into 1 send request and then just send one request instead of 4
```
local req = im.InteractionRequest(im.InteractionRequest.RequestType.READ, {})
req:merge(clusters.Thermostat.attributes.ControlSequenceOfOperation:read(device))
reg:merge(clusters.FanControl.attributes.FanModeSequence:read(device))
reg:merge(clusters.FanControl.attributes.WindSupport:read(device))
reg:merge(clusters.FanControl.attributes.RockSupport:read(device))
device:send(req)
```

Reply:
Good idea! That change is being handled [here](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1578) in a fanOscillationMode specific PR. All fanOscillationMode changes have been removed from this PR.